public/stylesheets/style2.css

```diff
@@ -0,0 +1,93 @@
+body {
```

Comment:
Do we need two separate files for CSS styles ?
Can we add this in the other styles file ?